yeonjae/programmers-1/문자열내맘대로정렬하기.js

```diff
@@ -0,0 +1,15 @@
+function solution(strings, n) {
+
+  strings.sort ((a,b) => {
+    const firstVal = a[n];
+    const secondVal = b[n];
+        
+    if (firstVal === secondVal) {
+      return (a > b) - (a < b);
+    } else {
+      return (firstVal > secondVal) - (firstVal < secondVal);
+    }
+  });
+  
+  return strings;
+}
```

Comment:
```suggestion
function solution(strings, n) {

  strings.sort((a, b) => {
    const firstVal = a[n];
    const secondVal = b[n];
        
    if (firstVal === secondVal) {
      return (a > b) - (a < b);
    } 
    return (firstVal > secondVal) - (firstVal < secondVal);
  });
  return strings;
}
```
  - 고생하셨습니다 👍 하연님과 로직이 비슷해서 하연님쪽 리뷰 한번 참고해보시고 풀이 주석 남겨주세요~ 🙆‍♂️
  - 함수 호출 시 그 다음 괄호는 띄어쓰기 안하는게 원칙입니다 🙆‍♂️ 그리고 `,` 다음에 띄어쓰기도 해주세요